mu-plugins/central-hub/tests/phpunit/unit/meta-data/remapCustomFieldsConfig.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Tests for the function remap_custom_fields_config().
+ *
+ * @package     spiralWebDb\centralHub\Tests\Unit\Metadata
+ * @since       1.3.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Unit\Metadata;
+
+use Brain\Monkey;
+use function spiralWebDB\Metadata\remap_custom_fields_config;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_RemapCustomFieldsConfig
+ *
+ * @package spiralWebDb\centralHub\Tests\Unit\Metadata
+ * @group   meta-data
+ */
+class Tests_RemapCustomFieldsConfig extends Test_Case {
+
+	/**
+	 * Prepare the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once CENTRAL_HUB_ROOT_DIR . '/src/meta-data/meta-box.php';
+	}
+
+	/**
+	 * Clean up the test environment after each test.
+	 */
+	public function tearDown() {
+		parent::tearDown();
+		$config = [];
```

Comment:
@rgadon107 The `tearDown()` method is not needed in this test case. Why?

`$config` is local to the `tearDown()` method, ie not available to the other methods. Since it's local to this method, it serves no purpose. Right? It's not a property.

Therefore, you can remove the `tearDown()` method.

Reply:
`tearDown()` method removed.